Britannia/events/BTWK_storyline_events.txt

```diff
@@ -0,0 +1,390 @@
+﻿namespace = BTWK_story_event
+
+################################
+# THE WINTER KING STORY EVENTS #
+################################
+
+# CURRENTLY JUST PORTS FROM CKII. AMENDMENTS TO MAKE THEM CLOSER TO THE STORY SHOULD BE COMPLETED AS REQUIRED.
+
+################################
+
+### UTHER'S DEATH and ARTHUR'S RETURN ###
+# BTWK_story_event.0001 = Autumn 480 AD. Uther Pendragon health is worsening...
+# BTWK_story_event.0002 = January 481 AD. If Uther still alive, he becomes incapable...
+# BTWK_story_event.0003 = Uther is Dead - Hidden Effects: Triggers EVTs .0004 
+# BTWK_story_event.0004 = Arthur Returns - Hidden Effects: Title Swaps; Triggers EVTs .0005 & .0006
+# BTWK_story_event.0005 = Uther is Dead global ping
+# BTWK_story_event.0006 = Unique event for player controlled mordred allowing tag switch to arthur
+# BTWK_story_event.0007 = Arthur introduction event
+
+### ARTHUR REGENCY CHAIN ###
+# BTWK_story_event.0008 = Regent decides to give power to Arthur
+# BTWK_story_event.0009 = Arthur accepts regency
+# BTWK_story_event.0010 = Notify Mordred
+# BTWK_story_event.0011 = Notify other vassals
+# BTWK_story_event.0012 = Set Flag
+
+
+############## UTHER'S DEATH and ARTHUR'S RETURN ##############
+
+
+BTWK_story_event.0001 = {  # Triggered by on_game_start on_action incase Uther's pneumonia is cured. Possibly make it non-hidden also. "Second bout of illness..."
+	title = BTWK_story_event.0001.t
+	desc = BTWK_story_event.0001.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = worry
+	}	
+	lower_left_portrait = {
+		character = character:5  # Mordred ap Mordred
+		animation = no
+	}
+	lower_right_portrait = {
+		character = character:2  # Merlin
+		animation = no
+	}
+	lower_center_portrait = {
+		character = character:20  # Tewdric
+		animation = no
+	}
+
+	
+	option = {
+		name = BTWK_story_event.0001.a
+		trigger = {
```

Comment:
Probably don't need these triggers for single-option events - I don't know how CK3 reacts if you don't have valid options (but in other games it's usually not good).

Reply:
I have removed the trigger, and instead added an "if-limit" block to on_game_start, similar in structure to the on_death change you suggested (checking instead for the global variable "utherillness" not existing, just to make sure it doesn't infinitely trigger)